<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanningSettings</class>
 <widget class="QWidget" name="PlanningSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Path Planning</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Planning Parameters</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Obstacle Expansion</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="obstExpRadSB"/>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Obstacle Penalty</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Bridge Test Res.</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Regular Grid Res.</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Bridge Length</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Connection Radius</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="bridgeTestResSB"/>
               </item>
               <item row="2" column="1">
                <widget class="QDoubleSpinBox" name="bridgeSegLenSB"/>
               </item>
               <item row="3" column="1">
                <widget class="QDoubleSpinBox" name="regGridResSB"/>
               </item>
               <item row="4" column="1">
                <widget class="QDoubleSpinBox" name="nodeConRadSB"/>
               </item>
               <item row="5" column="1">
                <widget class="QDoubleSpinBox" name="obstPenRadSB"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Planning Steps</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QCheckBox" name="bridgeTest">
                 <property name="text">
                  <string>Bridge Test</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="connectNodes">
                 <property name="text">
                  <string>Node Connection</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="expandObst">
                 <property name="text">
                  <string>Obstacle Expansion</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="regGrid">
                 <property name="text">
                  <string>Regular Grid Generation</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="obstPenalty">
                 <property name="text">
                  <string>Obstacle Penalty</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Display Settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QCheckBox" name="showSearchSpaceTree">
                 <property name="text">
                  <string>Show Search Space Tree</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showSearchTree">
                 <property name="text">
                  <string>Show Search Tree</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showPaths">
                 <property name="text">
                  <string>Show Generated Path</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showRobotTrail">
                 <property name="text">
                  <string>Show Robot Trail</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showSearchSpaceSamples">
                 <property name="text">
                  <string>Show Search Space Nodes</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Obstacle Avoidance</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Avoidance Method</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QRadioButton" name="noavoidRadBtn">
               <property name="text">
                <string>No avoidance (Local Controller)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="vfhRadBtn">
               <property name="text">
                <string>Nearness Diagram / VFH (Reactive Planner)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
